### DISCUSSION TITLE
Welcome to LaunchBar Discussions!

### DISCUSSION THREAD

**Discussion by Ptujec** [Ptujec/LaunchBar#7] [Announcements]
## 👋 Welcome!
 This is a place where you can:
  * Ask questions about how to use my actions
  * Share ideas for existing or new actions  

Don't forget to be kind ✌🏻
Cheers, Christian

**Comment by andrewcanion** [comment]
I moved away to Alfred a while ago, but your Actions tempt me back to Launchbar more than the core app!

**Reply by Ptujec** [reply]
Ha, thanks. I am sure you will find plenty of quality workflows for Alfred. Or Raycast, if you want to try something new. I love LBs concept, but I know it's not for everyone.

**Comment by andrewsardone** [comment]
Good idea to use GitHub Discussions! We need more LaunchBar community resources 👍

**Reply by Ptujec** [reply]
Thanks. I found it on accident. GitHub is a fitting place. Would be cool though if we could have one for all LaunchBar repos … not just mine.

**Reply by Ptujec** [reply]
There is also a reddit group https://www.reddit.com/r/LaunchBar/

**Comment by xtof41** [comment]
Hi, thank you for these actions !
Regarding the Deepl one : any possibility to chose other languages than the 5 available ?

**Reply by Ptujec** [reply]
1.3 is up now with all supported languages

**Comment by jonohunt** [comment]
How am I only just finding this here now?! 😊

**Reply by Ptujec** [reply]
🤷🏻‍♂️😊

**Comment by rlguibord** [comment]
Ptujec, I have used several of the actions for Launchbar that you have freely shared, most recently the pathway for accessing the 1Password CLI in Launchbar. I was thrilled to get this working as I had just begun experimenting with Alfred and already knew I didn’t want to go there. I am hoping there is a way for Ojective Development to pay you for your work. I will certainly be contributing.

**Reply by Ptujec** [reply]
Thank you for your kind words! 😊 It is always nice to hear when someone benefits from my actions. 
The guys at Objective Development don't owe me. I am just happy to know they are committed to their product. They also expressed their appreciation several times and listen to my feedback. It's all good.
